packages/example/src/ColorSchemeContext.ts

```diff
@@ -0,0 +1,29 @@
+/*
+  The MIT License
+
+  Copyright (c) 2017-2019 EclipseSource Munich
+  https://github.com/eclipsesource/jsonforms
```

Comment:
since this is a new file, the copyright notice does not need to include EclipseSource